### ISSUE TITLE
XFCE panel nm-applet VPN connecting and connected icons are broken.

### ISSUE THREAD

**Issue by James5979** [numixproject/numix-icon-theme#1025]
I think this may be a problem with overlays, similar to: https://github.com/numixproject/numix-icon-theme/issues/989 .  Took some screenshots to help:

Normal USB tethered connection, same as normal Ethernet connection for icons:
https://i.imgur.com/Q0vXJqU.png

VPN connecting:
https://i.imgur.com/I8cREj9.png

VPN connected:
https://i.imgur.com/EXc0z9F.png

The last two screenshots above show the broken looking nm-applet icons. Thanks for anything that you can do to remedy this.

**Comment by wa4557** [comment]
Well that's new. For the second screenshot is there an animation showing?

**Comment by James5979** [comment]
I made a .gif file of the animation of the icon when connecting to the VPN.

https://i.imgur.com/4HoDSxw.gifv

**Comment by wa4557** [comment]
I'm not sure we can do anything about this :( In order for this to work out, we would need to change the animations. Which would be very easy to do, however, the thing is Ubuntu for example is not using those overlays, so removing the wifi shape from the icons would show just emptiness. 

I'm not a big fan of those overlay icons, they just make everthing more complicated...

**Comment by James5979** [comment]
Yes, that would not be acceptable for Ubuntu users, so this leaves us in a bit of a pickle. As a suggestion, a possible way to overcome this would be to make a clone of the numix icon theme specifically for Xfce, but that would be overly complicating for just a single bug. The only possible way (maybe), would be to make an Xfce specific fixer script that an Xfce user could run to replace the broken animation with the correct annimation, but still not an elegant or concise solution. Still, thankyou for trying to resolve this issue and I hope one day Xfce would stop using overlays in their nm-applet. Keep up the good work, appreciate all that you guys do.

**Comment by jrb** [comment]
Unfortunately this issue isn't limited to just Xfce, nor is it only the VPN. When you throw additional Mobile Broadband into the mix it gets even messier as Numix doesn't yet have an LTE icon:

![numixlte](https://cloud.githubusercontent.com/assets/397735/18452158/aebbd3f4-78ee-11e6-927f-6c8731d9ac6d.png)

![numixltevpn](https://cloud.githubusercontent.com/assets/397735/18452163/b4564772-78ee-11e6-8a46-3528e3d82880.png)

After digging through the network-manger source a bit, it looks like the possible icons are:

```
Disconnected:
    nm-no-connection

Connecting (Wired, Wireless, Mobile)
    nm-stage%02d-connecting%02d

Wired:
    nm-device-wired

Wireless (client):
    nm-device-wireless || nm-signal-%d

Mobile:
    nm-wwan-tower
    nm-signal-%d
    if roaming:
        nm-mb-roam
    if tech:
    nm-tech-%s

VPN:
    nm-vpn-connecting%02d
    nm-vpn-active-lock
```

**Comment by wa4557** [comment]
Thanks. That's also related to #989. 

This is still completely stupid. Take mobile for example. There is an overlay with `nm-wwan-tower` `nm-signal-` and `nm-tech`. This is only possible if `nm-wwan-tower` or `nm-signal-%d` is transparent...
